### PR TITLE
[ADS-120] chore: deprecate react bootstrap

### PR DESCRIPTION
**Pull request by Harris-YWang**
## Description

- Deprecate `react-bootstrap` with replacing `<Modal />` with `<ActionPanel isModal />`.
- Refactor `<ListPicker />` component
- Also refactor `<ConfirmModal />` and `<UserListPicker />`
- All above changes are supposed to align with the old UI, styles and how to use it.

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

**Comment by Harris-YWang** [comment]
the feat is tested by @minhe86

**Review by xiaofan2406 — APPROVED** [review]

File: www/examples/Tab.mdx [hunk 1]
```diff
@@ -53,8 +53,7 @@ render(Example);
 
 <DesignNotes>
   Tabs are most commonly used with areas that require the user to switch between views. Each tab label is supported by
-  an icon. This is not a react-bootstrap component. However it implements the same API for switching tabs. Only the
-  props listed are supported.
+  an icon. However it implements the same API for switching tabs. Only the props listed are supported.
```
Comment (renrizzolo) [inline www/examples/Tab.mdx:56]: This needs to be reworded. "However it implements the same API for switching tabs." doesn't make sense any more. Maybe just remove that sentence?

Comment (Harris-YWang) [reply www/examples/Tab.mdx:56]: Cool, I will remove this line, thanks

**Review by renrizzolo — APPROVED** [review]
LGTM

**Comment by renrizzolo** [comment]
merging ...

**Pull request merged**